src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -34,6 +34,8 @@ namespace Microsoft.Terminal.Control
         void UpdateControlSettings(IControlSettings settings);
         void UpdateControlSettings(IControlSettings settings, IControlAppearance unfocusedAppearance);
 
+        Microsoft.Terminal.TerminalConnection.ITerminalConnection Connection { get ; set; };

```

Comment:
I think if it's both get and set, you can drop the explicit get/set, because that's the default for properties anyways.